streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -225,6 +277,7 @@ public void testSize() {
         store.put(2, "two");
         store.put(4, "four");
         store.put(5, "five");
+        store.flush();
```

Comment:
Note that before we didn't test caching stores, we test them now though hence this change was necessary.